pkg/controllermanager/apis/config/types.go

```diff
@@ -89,6 +93,24 @@ type ControllerManagerControllerConfiguration struct {
 	ShootHibernation ShootHibernationControllerConfiguration
 }
 
+// BackupBucketControllerConfiguration defines the configuration of the BackupBucket
+// controller.
+type BackupBucketControllerConfiguration struct {
+	// ConcurrentSyncs is the number of workers used for the controller to work on events.
+	ConcurrentSyncs int
+}
+
+// BackupEntryControllerConfiguration defines the configuration of the BackupEntry
+// controller.
+type BackupEntryControllerConfiguration struct {
+	// ConcurrentSyncs is the number of workers used for the controller to work on events.
+	ConcurrentSyncs int
+	// DeletionGracePeriodHours holds the period in number of days to delete the Backup Infrastructure after deletion timestamp is set.
+	// If value is set to 0 then the BackupEntryController will trigger deletion immediately.
+	// +optional
```

Comment:
@rfranzke `+optional` tag in this entire file should be removed as well right?

Reply:
Yes, thank you